src/test/cypress/support/pageobjects/assessment/CourseAssessmentDashboardPage.ts

```diff
@@ -21,4 +24,10 @@ export class CourseAssessmentDashboardPage {
     checkShowFinishedExercises() {
         cy.get('#field_showFinishedExercise').check();
     }
+
+    clickEvaluateQuizzes() {
```

Comment:
The previous version of this method is in cypress/support/pageobjects/exam/StudentExamManagementPage.ts, isn't it? Should we remove it from there?

Reply:
It didnt exist back when i made this PR but now it makes sense to do this 👍